ngraph/core/include/ngraph/runtime/reference/hswish.hpp

```diff
@@ -0,0 +1,38 @@
+//*****************************************************************************
```

Comment:
Please move reference implementation to `ngraph/core/reference`

Reply:
Done